utils/setup_samples.sh

```diff
@@ -5,12 +5,12 @@ ERA=$1
 TAG=$2
 [[ ! -z $3 ]] && WD=$3 || WD=$( pwd -P )
 # Samples Run2016
-basedir="/ceph/sbrommer/artus_ntuple/2016_samples"
```

Comment:
is this okay @harrypuuter ?

Reply:
i mean those are old anyways - but i will put the new ones there when they are ready

Reply:
Yeah well, when the new ones are tested and all, might as well push them to /ceph/htautau/2016 anyway, no?